packages/orbit-components/src/Card/index.tsx

```diff
@@ -99,52 +66,16 @@ const Card = ({
             onClose={onClose}
             header={header}
           />
-        </CardWrapper>
+        </div>
       )}
 
-      {children
-        ? React.Children.map(children, (item, key) => {
-            if (!item) return null;
-            const topRoundedBorder =
-              expandedSections.indexOf(key - 1) !== -1 || expandedSections.indexOf(key) !== -1;
-            const bottomRounderBorder =
-              expandedSections.indexOf(key + 1) !== -1 || expandedSections.indexOf(key) !== -1;
-
-            // This is used for the case when user wants to map sections and change their order
-            // related issue: #1005
-            // @ts-expect-error TODO
-            const index = Number(item.key) || key;
+      {!loading && children}
 
-            return (
-              <SectionProvider
-                value={{
-                  addSection,
-                  removeSection,
-                  roundedBorders: {
-                    top: topRoundedBorder,
-                    bottom: bottomRounderBorder,
-                  },
-                  index,
-                  noBorderTop: index === 0 && Boolean(title),
-                  isOpened: expandedSections.some(val => val === index),
-                }}
-              >
-                {loading ? (
-                  <CardWrapper noPadding>
-                    <Loading loading={loading} type="boxLoader">
-                      {renderSection(item)}
-                    </Loading>
-                  </CardWrapper>
-                ) : (
-                  renderSection(item)
-                )}
-              </SectionProvider>
-            );
-          })
-        : null}
-    </StyledCard>
+      {loading && (
+        <div className="border-x-[1px] border-b-[1px]">
```

Comment:
If this is replacing `<CardWrapper noPadding>` this is missing `py-md`. Also, when a border is of 1px, can't we ommit its custom value? And just be `border-x border-b`?

Reply:
adjusted, thank, TIL